packages/headless/src/app/commerce-engine/commerce-engine-configuration.ts

```diff
@@ -38,26 +39,52 @@ export const commerceEngineConfigurationSchema =
     }),
   });
 
+// TODO KIT-3244: Use a different sample organization
 export function getSampleCommerceEngineConfiguration(): CommerceEngineConfiguration {
   return {
-    organizationId: 'fashioncoveodemocomgzh7iep8',
-    // deepcode ignore HardcodedNonCryptoSecret: Public key freely available for our documentation
-    accessToken: 'xx149e3ec9-786f-4c6c-b64f-49a403b930de',
+    accessToken: 'xxc481d5de-16cb-4290-bd78-45345319d94c',
```

Comment:
I'm a bit surprised that we commit `accesToken`s. Probably expired anyway?

Reply:
No they aint: But some access token that are solely made for search are OK.